src/parser/formatting-evaluator-module.ts

```diff
@@ -168,9 +170,26 @@ export class FormattingEvaluatorModule extends BaseModule {
         element.remove();
         continue;
       }
-      this._updateTagCount(formatting, tagName, score);
+      if (tagName === "a") {
+        const url = element.getAttribute("href");
+        if (url) {
+          urlSet.add(url.split("#")[0]);
```

Comment:
I'm assuming this is for unique anchors. You should consider the query parameter as well. I think let's just consider them the same link with different query parameters.